ee/clickhouse/migrations/0025_remove_dollar_from_materialized_columns.py

```diff
@@ -0,0 +1,74 @@
+from functools import lru_cache
+
+import structlog
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.materialized_columns.columns import get_materialized_columns, materialized_column_name
+from posthog.constants import GROUP_TYPES_LIMIT
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+logger = structlog.get_logger(__name__)
+
+
+def alias_column(table, current_name, new_name):
+    sync_execute(
+        f"""
+        ALTER TABLE {table} ON CLUSTER '{CLICKHOUSE_CLUSTER}'
+        ADD COLUMN IF NOT EXISTS \"{new_name}\"
+        VARCHAR ALIAS \"{current_name}\"
+    """
+    )
+
+    comment = get_column_names(table)[current_name]
+    if len(comment) > 0:
+        comment_column(table, current_name, "")
+        comment_column(table, new_name, comment)
+
+    logger.info(
+        "Created an ALIAS for column containing $ sign.", table=table, current_name=current_name, new_name=new_name
+    )
+
+
+def alias_column_if_exists(table, current_name, new_name):
```

Comment:
We do this if and the IF NOT EXIST SQL to avoid hitting the db all the time, is that right?

Reply:
IF NOT EXISTS is for checking if the _new_ column exists, this method is for checking whether the _old_ column also exists (which it won't on new setups)

Reply:
Ah I see it's for the current name, ignore